### ISSUE TITLE
kerl build R13B Error!

### ISSUE THREAD

**Issue by onlypengsir123** [kerl/kerl#227]
hi:
    When building R13B with kerl in default, I got the error as follow. Who will answer my question, please? 

`[root@pengsir:kerl]# ./kerl build R13B R13B
Verifying archive checksum...
Checksum verified (6d8c256468a198458b9f08ba6aa1a384)
Extracting source code
Building Erlang/OTP R13B (R13B), please wait...
Build failed.
        -emulator /root/.kerl/builds/R13B/otp_src_R13B/lib/compiler/src/genop.tab beam/ops.tab
Can't use 'defined(@array)' (Maybe you should just omit the defined()?) at utils/beam_makeops line 1462.
x86_64-unknown-linux-gnu/Makefile:489: recipe for target 'x86_64-unknown-linux-gnu/opt/plain/beam_opcodes.h' failed
make[2]: *** [x86_64-unknown-linux-gnu/opt/plain/beam_opcodes.h] Error 255
make[2]: Leaving directory '/root/.kerl/builds/R13B/otp_src_R13B/erts/emulator'
/root/.kerl/builds/R13B/otp_src_R13B/make/run_make.mk:38: recipe for target 'generate' failed
make[1]: *** [generate] Error 2
make[1]: Leaving directory '/root/.kerl/builds/R13B/otp_src_R13B/erts/emulator'
Makefile:292: recipe for target 'depend' failed
make: *** [depend] Error 2

Please see /root/.kerl/builds/R13B/otp_build_R13B.log for full details.`

/root/.kerl/builds/R13B/otp_build_R13B.log as follow:
`cd erts/emulator && ERL_TOP=/root/.kerl/builds/R13B/otp_src_R13B make generate depend
make[1]: Entering directory '/root/.kerl/builds/R13B/otp_src_R13B/erts/emulator'
make -f x86_64-unknown-linux-gnu/Makefile generate
make[2]: Entering directory '/root/.kerl/builds/R13B/otp_src_R13B/erts/emulator'
/bin/mkdir -p x86_64-unknown-linux-gnu/opt/plain
/bin/mkdir -p obj/x86_64-unknown-linux-gnu/opt/plain
/bin/mkdir -p pcre/obj/x86_64-unknown-linux-gnu/opt
/bin/mkdir -p zlib/obj/x86_64-unknown-linux-gnu/opt
LANG=C /usr/bin/perl utils/beam_makeops -outdir x86_64-unknown-linux-gnu/opt/plain \
    -emulator /root/.kerl/builds/R13B/otp_src_R13B/lib/compiler/src/genop.tab beam/ops.tab
Can't use 'defined(@array)' (Maybe you should just omit the defined()?) at utils/beam_makeops line 1462.
x86_64-unknown-linux-gnu/Makefile:489: recipe for target 'x86_64-unknown-linux-gnu/opt/plain/beam_opcodes.h' failed
make[2]: *** [x86_64-unknown-linux-gnu/opt/plain/beam_opcodes.h] Error 255
make[2]: Leaving directory '/root/.kerl/builds/R13B/otp_src_R13B/erts/emulator'
/root/.kerl/builds/R13B/otp_src_R13B/make/run_make.mk:38: recipe for target 'generate' failed
make[1]: *** [generate] Error 2
make[1]: Leaving directory '/root/.kerl/builds/R13B/otp_src_R13B/erts/emulator'
Makefile:292: recipe for target 'depend' failed
make: *** [depend] Error 2`

Version:
     linux: ubuntu 16.04
     kerl: 1.4.2

**Comment by mrallen1** [comment]
As per our [OTP support policy](https://github.com/kerl/kerl#otp-support-policy) this is a best effort fix, but I *believe* you can get this to build by modifying kerl around [line 478](https://github.com/kerl/kerl/blob/master/kerl#L478) to include 13 as your build version, applying the 14 patch. 

It would look something like this:

```diff
diff --git a/kerl b/kerl
index b292ea3..29a449a 100755
--- a/kerl
+++ b/kerl
@@ -476,6 +476,9 @@ maybe_patch_all()
     perlver=$(get_perl_version)
     if [ "$perlver" -ge 22 ]; then
         case "$1" in
+            13)
+                apply_r14_beam_makeops_patch >> "$LOGFILE"
+                ;;
             14)
                 apply_r14_beam_makeops_patch >> "$LOGFILE"
                 ;;
```

**Comment by onlypengsir123** [comment]
Think you, now it wroks well. 
But I have found that the value of 'perlver' in kerl around line 475  is 24. If I does not modify 24 to 22, it also build failed, the error information as above.

**Comment by mrallen1** [comment]
Yeah we fixed that in #198. Glad it's working.
